### ISSUE TITLE
nodupe_basis doesn't show up in sr3 show and it's confusing

### ISSUE THREAD

**Issue by reidsunderland** [MetPX/sarracenia#1111]
There's three cases:

1) nodupe_basis **is not** set in the config --> sr3 show does not show nodupe_basis with it's default value (path)
2) nodupe_basis **is** set to `name` or `data` --> sr3 show does not show nodupe_basis
3) nodupe_basis **is** set to anything other than `name` or `data` (including `path`) --> sr3 show **does** show the value of nodupe_basis

Case 1 is because "path" seems to be an implied default, it gets changed by loading other plugins like nodupe.name or nodupe.data. 

Case 2 is because when you override the default with a valid alternative, config.py deletes the nodupe_basis attribute after loading the appropriate plugin: 

https://github.com/MetPX/sarracenia/blob/f5b8f18bf6962cb4c0275de1bd1a21a0588d4caf/sarracenia/config.py#L1870-L1876

Case 3 is because there's no else in the above code.

@andreleblanc11 and I were really confused by this until we looked at the code for a long time.

These are changes that I think would help:

1) defining nodupe_basis in default_options --> this should make `nodupe_basis: path` show up in sr3 show when the user doesn't set nodupe_basis in their config
2) removing the delattr lines, so if the user sets nodupe_basis to name or data, it will show up in sr3 show
3) Maybe: adding an else to catch cases where nodupe_basis is not path, data nor name. The delattr would be okay in this case.

**Comment by petersilva** [comment]
yeah... I remember trying to fix this a while back by innocently removing the delattr... and stuff broke.  It should definitely be done, it's just not as trivial as it should be.
